sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOIT.java

```diff
@@ -106,12 +105,6 @@ public void testWriteThenRead() {
     runRead();
   }
 
-  @Test
-  public void testWriteThenReadWithWhere() {
```

Comment:
This one was removed because the functionality is already tested in the unit tests via `withQuery`, and because ITs it could be easily subsumed by `testWriteThenReadWithWhere`